src/Pages/jiminkim/Main/Feeds/Feed.js

```diff
@@ -0,0 +1,154 @@
+                )}
+              </svg>
+            </li>
+            <li className="icon">
+              <svg
+                aria-label="댓글 달기"
+                fill="#262626"
+                height="24"
+                viewBox="0 0 48 48"
+                width="24"
+              >
+                <path
+                  clip-rule="evenodd"
+                  d="M47.5 46.1l-2.8-11c1.8-3.3 2.8-7.1 2.8-11.1C47.5 11 37 .5 24 .5S.5 11 .5 24 11 47.5 24 47.5c4 0 7.8-1 11.1-2.8l11 2.8c.8.2 1.6-.6 1.4-1.4zm-3-22.1c0 4-1 7-2.6 10-.2.4-.3.9-.2 1.4l2.1 8.4-8.3-2.1c-.5-.1-1-.1-1.4.2-1.8 1-5.2 2.6-10 2.6-11.4 0-20.6-9.2-20.6-20.5S12.7 3.5 24 3.5 44.5 12.7 44.5 24z"
+                  fill-rule="evenodd"
+                ></path>
+              </svg>
+            </li>
+            <li className="icon">
+              <svg
+                aria-label="게시물 공유"
+                fill="#262626"
+                height="24"
+                viewBox="0 0 48 48"
+                width="24"
+              >
+                <path d="M47.8 3.8c-.3-.5-.8-.8-1.3-.8h-45C.9 3.1.3 3.5.1 4S0 5.2.4 5.7l15.9 15.6 5.5 22.6c.1.6.6 1 1.2 1.1h.2c.5 0 1-.3 1.3-.7l23.2-39c.4-.4.4-1 .1-1.5zM5.2 6.1h35.5L18 18.7 5.2 6.1zm18.7 33.6l-4.4-18.4L42.4 8.6 23.9 39.7z"></path>
+              </svg>
+            </li>
+          </ul>
+          <ul className="list-style-none">
+            <li className="icon">
+              <svg
+                aria-label="저장"
+                fill="#262626"
+                height="24"
+                viewBox="0 0 48 48"
+                width="24"
+              >
+                <path d="M43.5 48c-.4 0-.8-.2-1.1-.4L24 29 5.6 47.6c-.4.4-1.1.6-1.6.3-.6-.2-1-.8-1-1.4v-45C3 .7 3.7 0 4.5 0h39c.8 0 1.5.7 1.5 1.5v45c0 .6-.4 1.2-.9 1.4-.2.1-.4.1-.6.1zM24 26c.8 0 1.6.3 2.2.9l15.8 16V3H6v39.9l15.8-16c.6-.6 1.4-.9 2.2-.9z"></path>
+              </svg>
+            </li>
+          </ul>
+        </div>
+        {feedLikePeeple.length > 0 ? (
+          <div className="feeds-likes">
+            <div className="profile-box small">
+              <img src="/images/jiminkim/profile_img.jpeg" alt="프로필 사진" />
+            </div>
+            <>
+              <span className="profile-box-desc bold">
+                {feedLikePeeple[0].id}
+              </span>{' '}
+              님 외&nbsp;
+              <span
+                class="bold like-count"
+                onClick={this.openFeedLikeModalBefore}
+              >
+                {feedLikePeeple.length}
+              </span>
+              명이 좋아합니다.
+            </>
+          </div>
+        ) : null}
+        <div className="feeds-content">
+          <span className="bold">{feedWriter}</span>
+          {isMoreContent ? (
+            this.state.isViewMorecontent ? (
+              feedContent.split('\n').map(line => (
+                <>
+                  {line}
```

Comment:
```suggestion
              feedContent.split('\n').map(line => (
                <Fragment key={line}>
                  {line}
```
fragment를 사용하더라도 map으로 반복해주는 요소들은 key prop이 필요합니다. 공식문서 참고해주세요! ([링크](https://ko.reactjs.org/docs/fragments.html))

Reply:
변경하였습니다. 감사합니다!